de_project/de_app/views.py

```diff
@@ -2,14 +2,16 @@
 
 from .models import RecordHandler
 
-from django.http import HttpResponse
-from django.http import JsonResponse
+from django.http import HttpResponse, HttpResponseNotFound
 
 
 def index(request):
     return HttpResponse("Hello, world!")
 
 def get_data_as_json_file(request, record_handler_name):
-    record_handler = RecordHandler.objects.get(name=record_handler_name)
+    try:
+        record_handler = RecordHandler.objects.get(name=record_handler_name)
+    except:
+        return HttpResponseNotFound()
```

Comment:
This isn't quite right. 404 should only be returned if [`RecordHandler.DoesNotExist`](https://docs.djangoproject.com/en/3.1/ref/models/class/#doesnotexist) is thrown, not if *any* exception is thrown.

Reply:
It is done. Good one!